src/generate-attribution.py

```diff
@@ -165,13 +165,13 @@ def get_region_data(code: str) -> dict:
         region_data["subdivision_code"] = code
         region_data["subdivision_name"] = subdivision.name if subdivision else SUBDIVISIONS[code]
 
-    # TODO backward compatibility for not yet adapted consumer code, remove eventually
-    region_data["region_code"] = region_data.get("subdivision_code", region_data["country_code"])
```

Comment:
It looks like Träwelling is parsing this, so they need to be contacted before this can be removed.

Reply:
I'll open an issue there.